### DISCUSSION TITLE
Can you manually extend a vtable?

### DISCUSSION THREAD

**Discussion by matcool** [NationalSecurityAgency/ghidra#3182] [Q&A]
After upgrading to Ghidra 10.0, one of my vtables got messed up and ends sooner than its supposed to, is there any way to fix it?
![image](https://user-images.githubusercontent.com/26722564/124353112-e4a05b80-dbda-11eb-84b0-bb27e6133649.png)
(the values after the vtable "stops" are valid function addresses that I had previously set just fine)

**Comment by astrelsky** [comment]
If you just want to change the data in the listing you can change the type to an array of a larger size.
